app/models/event.js

```diff
@@ -0,0 +1,34 @@
+var Sequelize = require('sequelize');
+
+var schema = {
+	payload: {
+		type: Sequelize.JSONB,
+		allowNull: false
+	},
+	type: {
```

Comment:
I don't think we ended up talking it through, but do we want explicitly different event types for inbound and outbound?

If so, this is fine, but if they are going to be the same from a "type" perspective then we should be sure to track whether events are inbound or outbound (and who the sender / recipients were)